src/drivers/heater/heater.cpp

```diff
@@ -339,8 +285,7 @@ int Heater::start()
 	update_params(true);
 	initialize_topics();
 
-	// Kick off the cycling. We can call it directly because we're already in the work queue context
-	cycle();
+	ScheduleDelayed(10000);
```

Comment:
Hi @dagar, I don't have hardware to test this, but by inspection I think this could just be a `ScheduleNow()`call.

This PR really simplifies things.  Nice work.

Reply:
Changed to ScheduleNow().